src/CommitmentsV2/SFA.DAS.CommitmentsV2/Data/Configuration/DraftApprenticeshipConfiguration.cs

```diff
@@ -9,6 +9,7 @@ public class DraftApprenticeshipConfiguration : IEntityTypeConfiguration<DraftAp
         public void Configure(EntityTypeBuilder<DraftApprenticeship> builder)
         {
             builder.HasBaseType<ApprenticeshipBase>();
+            builder.Property(e => e.Id).ValueGeneratedNever();
```

Comment:
Is this correct to add this? Wouldn't this prevent an Id being created for the newly inserted records?